test/test.js

```diff
@@ -91,7 +100,7 @@ describe('Room', () => {
     const creep = new Creep('attackunreserve');
     const config = room.getCreepConfig(creep);
     assert.equal(config.opts.memory.role, 'attackunreserve');
-    assert.deepEqual(config.body, ['attack', 'attack', 'attack', 'move', 'move', 'move', 'ranged_attack', 'heal', 'heal', 'move', 'move', 'move']);
+    assert.deepEqual(config.body, ['move', 'move', 'move', 'attack', 'move', 'attack', 'move', 'heal', 'heal', 'attack', 'move', 'ranged_attack']);
```

Comment:
because I do think if the situation is soo bad that the creep fight until the last part, then it might better be a fighting part...

maybe I be wrong.

Reply:
I think you be right. now I moved the healing parts back somehow.

Reply:
maybe I be wrong, but i guess it depends on the situation